src/parser/jobs/rpr/modules/ArcaneCircle.tsx

```diff
@@ -24,6 +24,9 @@ export class ArcaneCircle extends RaidBuffWindow {
 
 	override buffStatus = this.data.statuses.ARCANE_CIRCLE
 
+	// RPR should not evaluate Arcance Circle overwrites since it's also a gauge generator
+	override evaluateOverwrites = false
```

Comment:
I think this is a mistake. Napkin math follows:

```
PH = 1000p
50 shroud gauge ~= 313*5 = 1565p
Odds of losing a use of AC by holding it for 20s = (20/120) = 1/6
Avg. potency gain from overwriting another RPR's AC = (1000 + 1565)(1/6) = 427.5p
```

For a gain of 427.5p you're losing 3% on 8 players for 20s. That works out to around 1780 RPR-equivalent potency (healer/caster/ranged potency is worth more, tank potency is worth less) per player in 20s, which is easily surpassed by all jobs - even Glare spam for 20s is significantly more than this. Therefore on average, overwriting AC seems like a bad practice to recommend.

Thoughts?

Reply:
Since I'm an idiot and forgot that the whole thing was informational only, and didn't have suggestions attached to it, I'm now fine with this. Done!